yt/sample_data_registry.json

```
@@ -47,6 +47,12 @@
     "load_name": "grid1.nc",
     "url": "https://yt-project.org/data/CfRadialGrid.tar.gz"
   },
+  "ChollaSimple.tar.gz": {
+    "hash": "b9b73eda41a7a4e7d2e5b8e263f4230334d316f003da616648002f913c655c43",
```

tip: you can test if the hash is correct by trying to run
```python
import yt
yt.load_sample("ChollaSimple")
```
when the hash isn't correct, pooch raises an error showing both the hash read from this file and the actual one.

I was able to generate the same hash pooch reported with 
```
openssl sha256 ChollaSimple.tar.gz
```
```suggestion
    "hash": "aa277471f694f9de2e46c190efd8cd631c0694044aa29cb8431f127319774d73",
```